app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,6 @@
+class SessionsController < ApplicationController
+  def destroy
+    sign_out(current_user)
+    redirect_to root_path, notice: 'You have been signed out successfully.'
+  end
+end
```

Comment:
Nice job making the requested changes. However, further additions have created a bug which is preventing your app from functioning properly as shown below. Kindly ensure that all is working fine before submitting for approval.

The error message "The action 'new' could not be found for SessionsController" typically occurs when there is no new action defined in the SessionsController. The new action is responsible for rendering the login form.

To resolve this issue, you can follow these steps:

1. Open the SessionsController file (usually located in app/controllers/sessions_controller.rb).

2. Make sure that the new action is defined within the controller. It should look something like this:

        class SessionsController < ApplicationController
              def new
                  # Your code to render the login form
              end

              # Other actions such as 'create' or 'destroy'
          end

![Screenshot from 2023-06-30 13-07-22](https://github.com/Yosefgeda/blog-app/assets/65233860/bb2c4a1d-8dc9-460a-89d1-27e89d8dbb76)


![Screenshot from 2023-06-30 13-12-01](https://github.com/Yosefgeda/blog-app/assets/65233860/f4131428-b69f-4055-981b-52ac0af9a598)